apps/nextjs-app-router/src/app/[lang]/[[...path]]/layout.tsx

```diff
@@ -13,10 +14,15 @@ export default function RootLayout({
   return (
     <html lang={params.lang}>
       <head>
-        <DraftModeScript />
+        <DraftModeScript appOrigin={process.env.MAKESWIFT_APP_ORIGIN} />
       </head>
       <body>
-        <MakeswiftProvider>{children}</MakeswiftProvider>
+        <MakeswiftProvider
+          previewMode={draftMode().isEnabled}
+          locale={params.lang}
```

Comment:
Yes, we lifted up `MakeswiftHostApiClient` to `ReactRuntimeProvider`, so it can't use the snapshot's locale as the default anymore. It shouldn't be needing a default locale at all, but first we need to adjust the builder to always pass a locale when performing resource mutations.

See also https://github.com/makeswift/makeswift/pull/836/files#diff-701ba0ed420276bea2f613b55a022aecc9f08aca9e519242cab81445ee070c7dR206 below.

Reply:
Great. Do we have a comment on `MakeswiftProvider`'s props as well so we remember to remove it from there in addition to removing the middleware?